### PR TITLE
As a user i want to use different spell checkers for different languages (SDBELGA 88)

### PR DESCRIPTION
**Pull request by tomaskikutis**
SDBELGA-88

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "superdesk-core",
-  "version": "1.28.0",
+  "version": "1.29.0-rc.1",
```
Comment (petrjasek) [inline package.json]: I think it should be `1.30.0-dev` now

Comment (tomaskikutis) [reply package.json]: Do you want to merge it now?

Comment (petrjasek) [reply package.json]: not really :wink:

Comment (tomaskikutis) [reply package.json]: I don't think there's a point to touch the version number then. When you decide you want to merge it, you can push an empty version bump commit to master and merge all the PRs which will inherit the new version number if `package.json` hasn't been modified in the PR.

**Pull request merged**